roles/etcd/templates/etcd-docker.service.j2

```diff
@@ -6,7 +6,7 @@ After=docker.service
 [Service]
 User=root
 PermissionsStartOnly=true
-EnvironmentFile=/etc/etcd.env
+EnvironmentFile=-/etc/etcd.env
```

Comment:
This prevents etcd from going into a hard failure state due to missing environmentfile and blocks service restart later when the file is available.